boards/lilygo/ttgo_lora32/ttgo_lora32-pinctrl.dtsi

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Lothar Felten <lothar.felten@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32-gpio-sigmap.h>
+
+&pinctrl {
+
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO1>;
+			output-high;
+		};
+		group2 {
+			pinmux = <UART0_RX_GPIO3>;
+			bias-pull-up;
+		};
+	};
+
```

Comment:
Would you mind to add SD via SPI support as well?
This seemed to work for me at least (in combination with the next suggestion):
```suggestion

	spim2_default: spim2_default {
		group1 {
			pinmux = <SPIM2_MISO_GPIO2>,
				 <SPIM2_SCLK_GPIO14>,
				 <SPIM2_CSEL_GPIO13>;
		};
		group2 {
			pinmux = <SPIM2_MOSI_GPIO15>;
		};
	};

```